app/graphql/types/customer/customer.gql

```diff
@@ -13,18 +22,13 @@ type CustomerStatus {
 }
 
 type AuthenticationQuestions {
```

Comment:
Maybe this should be `CustomerAuthenticationQuestions` to follow the same convention as `CustomerInfo` etc

Reply:
updated